tests/mlflow/Makefile

```
@@ -47,21 +49,54 @@ start-mlflow: install-mlflow ## Run MLflow server locally in the background
 	 MLFLOW_BACKEND_STORE_URI=$(MLFLOW_BACKEND_STORE_URI) \
 	 MLFLOW_DEFAULT_ARTIFACT_ROOT=$(MLFLOW_DEFAULT_ARTIFACT_ROOT) \
 	 MLFLOW_ENABLE_WORKSPACES=$(MLFLOW_ENABLE_WORKSPACES) \
+	 MLFLOW_LOG_FILE=$(MLFLOW_LOG_FILE) \
 	 VENV_DIR=$(VENV_DIR) \
 	 ./scripts/run_mlflow.sh
 	@echo "   MLflow server started in background. Use 'make stop-mlflow' to stop it."
 	@echo ""
 	@echo "   To use the server in your tests:"
 	@echo "   export MLFLOW_TRACKING_URI=$(MLFLOW_TRACKING_URI)"
+
+start-mlflow-with-logs: start-mlflow ## Run MLflow server locally in the background and tail the log file
```

checkmake (0.3.2)

[warning] Target body for "start-mlflow-with-logs" exceeds allowed length of 5 lines (32).

(maxbodylength)